src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -227,6 +227,11 @@ public override void OnActionExecuted(ActionExecutedContext context)
                 context.Result = CreateOperationOutcomeResult(formatException.Message, OperationOutcome.IssueSeverity.Error, OperationOutcome.IssueType.Invalid, HttpStatusCode.BadRequest);
                 context.ExceptionHandled = true;
             }
+            else if (context.Exception is System.OperationCanceledException operationCanceledException)
+            {
+                context.Result = CreateOperationOutcomeResult(operationCanceledException.Message, OperationOutcome.IssueSeverity.Error, OperationOutcome.IssueType.Timeout, HttpStatusCode.RequestTimeout);
```

Comment:
We should return a more generic message, not these details

Reply:
@brendankowitz Thanks for the review. Would you mind giving some guidance on what the generic message should be? Would a message such as "An operation was canceled" suffice?

Reply:
Adding @EXPEkesheth for thoughts.

I think we should have something along the lines of `The attempted operation was cancelled, please retry the request. If you believe you have received this message in error, contact Customer Support.`